pandas/core/groupby.py

```diff
@@ -1487,7 +1487,7 @@ def wrapper(*args, **kwargs):
                                       (how, dtype_str))
         return func, dtype_str
 
-    def aggregate(self, values, how, axis=0):
+    def aggregate(self, values, how, axis=0, is_datetime=False):
 
```

Comment:
you can't put a flag like this here, its very odd. There are already introspection determinations in the grouper objects.